x/ccv/provider/types/codec.go

```diff
@@ -31,7 +31,9 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations(
 		(*sdk.Msg)(nil),
 		&MsgAssignConsumerKey{},
-		&MsgConsumerAddition{},
+		&MsgConsumerAddition{}, //TOOD @permissionless: replace with below
```

Comment:
I think we can remove this comment from this PR, because permissionless is not in `main` yet.